src/components/about.js

```diff
@@ -0,0 +1,68 @@
+import styled from '@emotion/styled';
+import Img from 'gatsby-image';
+import PropTypes from 'prop-types';
+import React from 'react';
+import External from '../assets/external.svg';
+import Github from '../assets/github.svg';
+import { mq } from './_shared/media';
+import { StyledHeading } from './_shared/styled-heading';
+import { contentBox, flexEnd } from './_shared/styled-mixins';
+import { StyledSection } from './_shared/styled-section';
+
+const StyledImageContainer = styled.section`
```

Comment:
we should extract that StyledImageContainer thing. We can create a follow up task though